src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMappingTest.java

```diff
@@ -524,9 +520,10 @@ void buildOverrideHearingLocations() {
                 .build());
     }
 
-    void enableAdjournmentFlag() {
+    void enableAdjournmentFlagAndSetAdjournmentToInProgress() {
```

Comment:
Given that this is only used three times, and is a small method, it is preferable to delete and have the lines directly in each relevant test. I have toned down the amount of refactoring I have been doing, especially in tests. 

Echoing Lucas' advice, tests should be clear what they are testing, and self-contained. Avoid extracting setup into helper methods, because it can lead to requiring tests for your tests! Even if it leads to some repetition, it is considered better to have each setup item independently stated in each test. 

If this would lead to having a large amount of repetition across many tests, then you should consider making the test class abstract with common setup, and extending it into separate classes with their own @BeforeEach for the unique setup. The BeforeEach method should ideally be the only "helper" which contains common setup code. 

Use your judgment when extending classes would be appropriate vs having a little repetition in the same class. For your case here, a little repetition is fine.

Reply:
deleted and put in relevant tests